CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/docs/scatter3d.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Arrange By Scatter 3D</title>
+        <link rel="stylesheet" href="nbdocs://au.gov.asd.tac.constellation.preferences/au/gov/asd/tac/constellation/preferences/constellation.css" type="text/css">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    </head>
+    <body>
+        <h2>Arrange By Scatter 3D</h2>
+        <p>
+            Arrange all the nodes as a 3D Scatter Gram.<BR>This is a 3D arrangement.
+        </P>
```

Comment:
Minor but probably be good to make this p and above br lowercase

Reply:
I'm pretty sure I copied it from the Grid one. Will change